### DISCUSSION TITLE
设置学位论文独创性声明和关于论文使用授权的说明页扫描文件路径问题？

### DISCUSSION THREAD

**Discussion by suppent** [note286/xduts#142] [Q&A] [answered]
![image](https://github.com/note286/xduts/assets/21963812/a17f511c-8a59-484f-84a1-fa29b4ac5617)
使用这个语法导入扫描文件，编译后，签名页没有替换成功。
这是我在配置里写的格式：statement-scan={sign.pdf}。

**Comment by note286** [comment, marked as answer]
无法复现你的问题。

```latex
\documentclass{xdupgthesis}
\xdusetup{info/statement-scan={example-image-duck-portrait.pdf}}
\begin{document}
\end{document}
```

![image](https://github.com/note286/xduts/assets/16207292/ca25d412-aa49-4666-94e0-6b595466e3ba)

**Reply by sikouhjw** [reply]
鸭鸭可爱捏
